ci/buildspec_main.yml

```diff
@@ -0,0 +1,28 @@
+# AWS CodeBuild config
+# This file is referenced in the infra repo as a CodeBuild buildspec target.
+
+# These 3 variables are set in the codebuild project definition's environment secion.
+# ECR_REGION
+# ECR_ACCOUNT_ID
+# ECR_NAMESPACE
+
+version: 0.2
+env:
+  shell: bash
+phases:
+  install:
+    runtime-versions:
+      nodejs: 16
+  pre_build:
+    commands:
+      - aws ecr get-login-password --region "${ECR_REGION}" | docker login --username AWS --password-stdin "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com"
+
+  build:
+    commands:
+      # Backend
+      - docker build backend/core -f build/docker/Dockerfile.backend --target tests
```

Comment:
Just a note that this won't actually be enough to run the tests.  You would need to do something like this:

```
docker build backend/core -f build/docker/Dockerfile.backend --target tests -t doorway/backend:test .
docker run doorway/backend:test yarn test:cov
docker run doorway/backend:test yarn test:<other_tests>
docker build backend/core -f build/docker/Dockerfile.backend --target run -t doorway/backend:run .
```

The e2e tests will need a database to connect to, but the unit tests should run without any additional dependencies.